actions/parse-and-mask-test-secrets/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "parse-and-mask-test-secrets",
+  "version": "0.0.1",
```

Comment:
Leave this as `0.0.0`. This  will be bumped and tagged by `changesets`

Run `pnpm changeset add` to add a changeset for this action.